server/src/main/java/org/elasticsearch/index/get/ShardGetService.java

```diff
@@ -131,7 +130,7 @@ public GetResult get(
         );
     }
 
-    private GetResult get(
+    private GetResult doGet(
```

Comment:
nit I see the rest of the class uses `inner` instead of `do` for private functions, feel free to see if renaming to `innerGet` might be more conforming.

Reply:
I wanted to make a distinction with the other innerGetAndFetch method.